src/backend/api/Fusion.Resources.Logic/Requests/Commands/ResourceAllocationRequest/Allocation/JointVentureAllocationRequestStarted.cs

```diff
@@ -56,10 +59,11 @@ private void ValidateWorkflow(DbResourceAllocationRequest request)
                             s.AddFailure("proposedPerson", "Must provide a person to be assigned the position"));
                 }
 
-                private ValueTask<bool> AssignRequestToResourceOwnerAsync()
+                private async ValueTask<bool> AssignRequestToResourceOwnerAsync(DbResourceAllocationRequest request)
                 {
-
-                    return new ValueTask<bool>(false);
+                    var department = await router.RouteAsync(request, CancellationToken.None);
```

Comment:
Should not update this. Add this empty default failing method as a placeholder as the step was sort of unclarified from the spec.. But now it is more clear that the JV workflows should never have an approval step. 

But fixing the JV workflow should be done in a separate pr anyways..